apps/06Checkers.js

```diff
@@ -49,15 +55,59 @@ function Board() {
         console.log(string);
     }
 
-    // Your code here
+    this.checkers = [];
+
+    this.createCheckers = function() {
+        var white = [[0, 1], [0, 3], [0, 5], [0, 7],
+        [1, 0], [1, 2], [1, 4], [1, 6],
+        [2, 1], [2, 3], [2, 5], [2, 7]];
+        var black = [[5, 0], [5, 2], [5, 4], [5, 6],
+        [6, 1], [6, 3], [6, 5], [6, 7],
+        [7, 0], [7, 2], [7, 4], [7, 6]];
+
+        for (var i=0; i <= 11; i++) {
+            this.grid[white[i][0]][white[i][1]] = new Checker('white');
+            this.grid[black[i][0]][black[i][1]] = new Checker('black');
+
+            this.checkers.push(this.grid[black[i][0]][black[i][1]]);
+            this.checkers.push(this.grid[white[i][0]][white[i][1]]);
+        }
+    }
+
+    this.selectChecker = function(row, column) {
+        return this.grid[row][column];
+    }
+
+
+    this.killChecker = function(position) {
+        var stringIt = position.toString();
+        var midpoint = stringIt.split("");
+        var kill = this.checkers.indexOf(this.selectChecker(midpoint[0], midpoint[1]));
```

Comment:
This is a little tough to read. May be better to break it into a few steps, such as getting the checker at the midpoint and assigning it to a variable, and then getting the index of that checker.

Also remember variable names should clearly convey what is contained within the variable. While kill somewhat gets the point of the variable, something like killIndex or indexOfCheckerToKill.